test/dns_dnscmd/dnscmd_test.rb

```diff
@@ -3,61 +3,70 @@
 require 'dns_dnscmd/dns_dnscmd_main'
 
 class DnsCmdTest < Test::Unit::TestCase
+  def setup
+    Proxy::Dns::Plugin.load_test_settings(
+        :dns_server => 'a_server',
+        :dns_zones => ['bar.domain.local', 'domain.local', '33.168.192.in-addr.arpa', '168.192.in-addr.arpa'])
+  end
+
   def test_dnscmd_provider_initialization
```

Comment:
> However, plugin initialization code is redundant, could you remove it?
Already tried this, though the unit fails if I do so. I'll need to loo at it.

Can you help me out what method I should use best?